### PR TITLE
Drupal 7 getLoginURL Fix

### PR DESCRIPTION
**Pull request by garethdylan361**
Drupal 7 getLoginURL was not supplying the base url.

### CODE REVIEW

File: CRM/Utils/System/Drupal.php [hunk 1]
```diff
@@ -163,7 +163,10 @@ public static function checkUserNameEmailExists(&$params, &$errors, $emailName =
    */
   public function getLoginURL($destination = '') {
     $query = $destination ? array('destination' => $destination) : array();
-    return url('user', array('query' => $query), TRUE);
+   	$config = CRM_Core_Config::singleton();
```
Comment (seamuslee001) [inline CRM/Utils/System/Drupal.php]: I think it could be improved with $baseUrl = CRM_Core_Config::singleton()->userFrameworkBaseURL;

**Comment by garethdylan361** [comment]
Made improvements. Thank you for the input!

File: CRM/Utils/System/Drupal.php (hunk 1 above)
Comment (garethdylan361) [inline CRM/Utils/System/Drupal.php]: made improvements. Thank you for the input!

**Comment by totten** [comment]
jenkins, ok to test

**Comment by garethdylan361** [comment]
replaced tabs with space indents

**Review by colemanw — CHANGES_REQUESTED** [review]
This appears to be passing a complete `http://...` url string into the drupal `url` function. I'm not sure that will work with clean urls disabled. I suspect the better way to do it would be by setting the `absolute` flag to true. Note that the function was being incorrectly used (a third param `TRUE` was being passed in but it only accepts 2 params.

File: CRM/Utils/System/Drupal.php (hunk 1 above)
Comment (eileenmcnaughton) [inline CRM/Utils/System/Drupal.php]: Actually - sorry to come late to the party but - shouldn't we be passing the 'absolute' param to the drupal function?

https://api.drupal.org/api/drupal/includes!common.inc/function/url/7.x

**Comment by colemanw** [comment]
@garethdylan361 can you please check the above?

**Comment by jitendrapurohit** [comment]
@colemanw @garethdylan361 I've raised a new PR at https://github.com/civicrm/civicrm-core/pull/9479. Also checked by enabling/disabling clean urls and it worked fine for me.

**Pull request closed without merging**